electron-app/src/components/settings/api-keys/ExchangeSettings.vue

```diff
@@ -68,6 +68,12 @@
             :type="showKey ? 'text' : 'password'"
             @click:append="showKey = !showKey"
           ></v-text-field>
+          <v-select
+            v-model="selectedKrakenAccountType"
+            class=""
```

Comment:
I did not really know what to put in the class here. @kelsos what do you normally use here?

Reply:
something like `exchange-settings__fields__kraken-account-type`

Reply:
@kelsos I see. But why?What does it achieve? Is it just a convention for class name in vue so that you can optionally customize its css?